lib/include/jxl/cms_interface.h

```diff
@@ -64,11 +64,20 @@ typedef struct {
  * @param output_profile the colorspace to which JxlCmsInterface::run should
  *        convert the input data.
  * @param intensity_target for colorspaces where luminance is relative
- *        (essentially: not PQ), indicates the luminance of (1, 1, 1). This is
- *        only useful for conversions between a relative luminance colorspace
- *        and an absolute luminance one, in either direction.
- *        @p intensity_target cd/m² in the absolute colorspace should map to and
- *        from (1, 1, 1) in the relative one.
+ *        (essentially: not PQ), indicates the luminance at which (1, 1, 1) will
+ *        be displayed. This is useful for conversions between a relative
+ *        luminance colorspace and an absolute luminance one, in either
+ *        direction: @p intensity_target cd/m² in the absolute colorspace should
+ *        map to and from (1, 1, 1) in the relative one.  
+ *        It is also used for conversions to and from HLG, as it is
+ *        scene-referred while other colorspaces are assumed to be
+ *        display-referred. That is, conversions from HLG should apply the OOTF
```

Comment:
When converting from PQ to HLG (or vice versa), is the intensity_target value then used for both purposes at the same time?

Reply:
Yes indeed. Taking PQ->HLG as an example: it is first used in the PQ->linear part to determine which part of the PQ range is mapped to 0-1, and then in linear->HLG to determine the gamma used in the inverse OOTF before the OETF. The end result is an HLG signal which, if displayed on a monitor with a peak luminance of `intensity_target`, would look the same as the PQ signal (although it wouldn’t scale in the same way to different peak luminances).

Conversely, HLG->PQ does: HLG ---[inverse OETF]--> 0-1 linear scene light ---[OOTF (`intensity_target`)]--> 0-1 linear display light ---[scaling (`intensity_target`) and PQ inverse EOTF]--> PQ, resulting in a PQ signal representing how the HLG signal would look on a monitor with a peak luminance of `intensity_target`.